doc/configuration.rst

```diff
@@ -616,6 +616,19 @@ Publishing configuration
 
        confluence_publish_postfix = '-postfix'
 
+    Postfixes can include placeholders. These placeholders are filled using the
+    format method so formatting types can be used.
+
+    Currently supported placeholders
+
+    * {hash} - Create a reproduceable hash given the title and location based 
+               from the project root. Using this placeholder provides an option
+               for allowing pages with the same title to be pushed to the same 
+               confluence space without needing to manually add an index 
+               to the title
+
+    Note, pages with the same title must have different parent pages.
```

Comment:
> [Pages] with the same title must have different parent pages.

I do not believe this note is applicable anymore (correct me if I am wrong here).